src/reducers/profileReducer.js

```diff
@@ -0,0 +1,28 @@
+import types from '../actions/actionTypes';
+
+export const initialState = {
+  isLoading: false,
+  errors: {
+    status: false,
+    message: '',
+    response: {},
+  },
+  profile: {},
+};
+
+export default (state = initialState, action = {}) => {
+  switch (action.type) {
+  case types.LOGOUT:
+    return {
+      ...state,
+      profile: {}
+    };
+  case types.LOGGED_IN_USER_PROFILE:
+    return {
+      ...state,
+      profile: action.payload
```

Comment:
can you make this `userProfile` so make it compatible with what I am using for the profile detail of the logged-in user